src/agent/Core/SpawningKit/UserSwitchingRules.h

```diff
@@ -56,6 +56,9 @@ struct UserSwitchingInfo {
 	gid_t gid;
 	int ngroups;
 	shared_array<gid_t> gidset;
+
+	struct passwd lve_userpwd, *lve_userpwd_complete;
```

Comment:
Please write field names and variable names in camelCase, as per the [Passenger C++ coding style](https://github.com/phusion/passenger/blob/stable-5.0/CONTRIBUTING.md#cxx_coding_style):

    struct passwd lveUserPwd, *lveUserPwdComplete;